conf.d/20_output_kubernetes_audit_elasticsearch.conf

```diff
@@ -30,7 +30,7 @@ filter {
 
     # Extract timestamp of the acutal event and overwrite @timestamp
     date {
-      match => [ "[audit_json][timestamp]", "ISO8601" ]
+      match => [ "[audit_json][stageTimestamp]", "ISO8601" ]
```

Comment:
Should rather be `requestReceivedTimestamp` as we mostly interested of when request hit the api.

Reply:
Actually you were right in the original PR. There can be multiple events in the log for the same API call and it's attributed to stage change. So please revert back to what you did initially. Thanks.

Reply:
Thanks I have changed it back

Reply:
👍